a3fe/configuration/engine_config.py

```diff
@@ -15,78 +15,53 @@
     ValidationInfo as _ValidationInfo
 )
 from ._engine_runner_config import EngineRunnerConfig as _EngineRunnerConfig
+from ..run.enums import LegType as _LegType, StageType as _StageType
 
-def _get_default_lambda_array(leg: str = "BOUND", stage: str = "VANISH") -> _List[float]:
-    
-    default_values = {
-        "BOUND": {
-            "RESTRAIN": [0.0, 1.0],
-            "DISCHARGE": [0.0, 0.291, 0.54, 0.776, 1.0],
-            "VANISH": [
+class SomdConfig(_EngineRunnerConfig, _BaseModel):
+    """
+    Pydantic model for holding SOMD engine configuration.
+    """ 
+    default_lambda_values: _Dict[_LegType, _Dict[_StageType, _List[float]]] = {
```

Comment:
Please remove from here and move into sysprepconfig (specifically the SomdSystemPreparationConfig)